nautobot/dcim/choices.py

```diff
@@ -762,7 +865,7 @@ class InterfaceTypeChoices(ChoiceSet):
     TYPE_16GFC_SFP_PLUS = "16gfc-sfpp"
     TYPE_32GFC_SFP28 = "32gfc-sfp28"
     TYPE_64GFC_QSFP_PLUS = "64gfc-qsfpp"
-    TYPE_128GFC_QSFP28 = "128gfc-sfp28"
+    TYPE_128GFC_QSFP28 = "128gfc-qsfp28"
```

Comment:
If I'm not mistaken this will cause data issues for those that would have the previous value.

Reply:
right, that's the string that ends up in the database. 

What would be the optimal solution? should I put it back or make a migration?

Reply:
Put it back for now. A migration right now in `develop` for something like this is overkill. It's not like there is a `TYPE_128GFC_SFP28` and `TYPE_128GFC_QSFP28` where the choice set is having a hard time picking between the two. In practice, the user has no significant impact, other than what they see in the API.

I could be persuaded otherwise @nautobot/core

Reply:
FWIW here's the commit in netbox that made this change and included the necessary corresponding data migration: https://github.com/netbox-community/netbox/commit/b7b5a5788f64e8b2a89b8153fd4a2554cb0f647d.

We *might* want to actually make the change and migration in 2.0, as my understanding is that `qsfp` is more correct in the case of this interface type?